include/hal/library/responder/key_pair_info.h

```diff
@@ -0,0 +1,74 @@
+/**
+ *  Copyright Notice:
+ *  Copyright 2024 DMTF. All rights reserved.
+ *  License: BSD 3-Clause License. For full text see link: https://github.com/DMTF/libspdm/blob/main/LICENSE.md
+ **/
+
+#ifndef RESPONDER_KEY_PAIR_INFO_H
+#define RESPONDER_KEY_PAIR_INFO_H
+
+#include "hal/base.h"
+#include "internal/libspdm_lib_config.h"
+#include "industry_standard/spdm.h"
+
+#if (LIBSPDM_ENABLE_CAPABILITY_GET_KEY_PAIR_INFO_CAP || \
```

Comment:
But the libspdm_rsp_set_key_pair_info_ack will call the `libspdm_read_key_pair_info` to get the capbilities. Thanks.

Reply:
`GET_KEY_PAIR_INFO_CAP` is required to be set if `SET_KEY_PAIR_INFO_CAP` is set. If `SET_KEY_PAIR_INFO_CAP` is set but `GET_KEY_PAIR_INFO_CAP` isn't, then it would result in a compilation error and that is fine.

Reply:
agree.